spec/index.md

```diff
@@ -5410,21 +5423,23 @@ We define an auxiliary method that handles calls from composite query methods by
                   status = simple_status(S.canister_status[Canister_id]);
                   canister_version = S.canister_version[Canister_id];
                 }
+      if liquid_balance(
```

Comment:
Does this condition really take precedence over checking if the canister is running etc.?

Reply:
The implementation is completely different, see `fn evaluate_query_call_graph` I've seen this and raised this with the team. We've decided that the exact error order is hard to express and not that important.

Reply:
But I'll move the `liquid_balance` closer to it's actual location in the code...

Reply:
> We've decided that the exact error order is hard to express and not that important.

In that case, we should consider leaving it out from the spec.

Reply:
We've discussed this option as well. This high-level algorithm might be still useful, but I don't have a strong opinion...

Reply:
Let's keep this as is.